handler.go

```diff
@@ -165,3 +186,50 @@ func WithHandler(new Handler) HandlerOption[*DefaultHandler] {
 		}
 	}
 }
+
+// WithSafeFuncs enables safe function calls in a DefaultHandler. When safe functions
+// are enabled, the handler will wrap all functions with a safe wrapper that logs any
+// errors that occur during function execution without interrupting the execution of
+// the template.
+//
+// To use a safe function, prepend `safe` to the original function name,
+// example: `safeOriginalFuncName` instead of `originalFuncName`.
+func WithSafeFuncs(enabled bool) HandlerOption[*DefaultHandler] {
+	return func(dh *DefaultHandler) {
+		dh.wantSafeFuncs = enabled
+	}
+}
+
+// safeWrapper create a safe wrapper function that calls the original function
+// and logs any errors that occur during the function call without interrupting
+// the execution of the template.
+func (dh *DefaultHandler) safeWrapper(functionName string, fn any) wrappedFunc {
+	return func(args ...any) (any, error) {
+		out, err := runtime.SafeCall(fn, args...)
+		if err != nil {
+			dh.Logger().With("function", functionName, "error", err).Error("function call failed")
+		}
+		return out, nil
+	}
+}
+
+// safeFuncName generates a safe function name by prepending "safe" to the original
+// function name and capitalizing the first letter of the function name.
+//
+// Example:
+//
+//	originalFuncName -> SafeOriginalFuncName
+func safeFuncName(name string) string {
+	if name == "" {
+		return ""
+	}
+
+	var b gostrings.Builder
+	b.Grow(len(name) + 4)
+
+	b.WriteString("safe")
+	b.WriteString(gostrings.ToUpper(string(name[0])))
+	b.WriteString(name[1:])
```

Comment:
Isn't strings.Title you are looking for?

Assuming you have no space in your method, it would work, no?


```suggestion	b.WriteString(gostrings.Title(string(name)))
```

Reply:
Changed to 
```go
cases.Title(language.Und, cases.NoLower).String(name)
``` 

due to the fact ` strings.Title` are deprecated since golang 1.18